multi/gr716-multi/adc.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Phoenix-RTOS
+ *
+ * GR716 ADC driver
+ *
+ * Copyright 2023 Phoenix Systems
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+#ifndef _MULTI_ADC_H_
+#define _MULTI_ADC_H_
+
+
+#include <sys/msg.h>
+
+
+extern void adc_handleMsg(msg_t *msg, int dev);
+
+
+extern int adc_init(void);
```

Comment:
Lets phase out these `extern`s
```suggestion
void adc_handleMsg(msg_t *msg, int dev);


int adc_init(void);
```